Launcher/Classes/ViewControllers/SettingsViewController.swift

```diff
@@ -36,10 +36,14 @@ class SettingsViewController: NSViewController {
             (linkField4, linkDescriptionField4)
         ]
         
-        if let cucumberProfile = applicationStateHandler.cucumberProfile {
+        if let cucumberProfile = plistOperations.readValues(forKey: Constants.Keys.cucumberProfileInfo).first {
```

Comment:
What about keeping this logic in the `ApplicationStateHandler`?

Reply:
I don't get that? What I mean is that you can just do `applicationStateHandler.cucumberProfile` in which internally (so in the `StateHandler`) it does `readValues` instead of reading from `UserDefaults`. @q231950, thoughts?

Reply:
#128 has been created to tackle this 👍